tests/qnn/test_keras.py

```diff
@@ -18,7 +18,7 @@
 import pytest
 
 import pennylane as qml
-from pennylane.qnn.keras import KerasLayer
+KerasLayer = qml.qnn.keras.KerasLayer
```

Comment:
This is to ensure that we hit [this](https://github.com/PennyLaneAI/pennylane/blob/61259e0ecae52b1c2d5463974652c18faf34acf1/pennylane/qnn/__init__.py#L46) line.

Reply:
😆